services/README.md

```diff
@@ -11,6 +11,5 @@ The services we use. Some we pay for, others we mooch off. We love them all.
 * **[Harvest](harvest)**
 * **[InVision](invision)**
 * **[SendGrid](sendgrid)**
-* **[Slack](slack)**
```

Comment:
Thanks, Cat - would it make sense to still mention that we use Slack, but our docs are in Drive now? Instead of removing the reference altogether?

Reply:
Oh yeah maybe! A lot of things in this list are very outdated. We'll need a rework in the future, I think it's on @kaseybon's long ToDo list.